R/utilities.R

```diff
@@ -517,6 +517,12 @@ broom_confint_terms <- function(x, ...) {
     rownames(ci) <- names(coef(x))[1]
   }
 
+  # Handle `coeftest` one-dimensional case
```

Comment:
Rather than dropping this code into the `broom_confint_terms()` helper, could you situate it into the `tidy.coeftest()` method itself? After calling `broom_confint_terms()` there, we still have access to `rownames(x)` and the result of this function. This will prevent tidier-specific code from leaking into more general tools.

Reply:
Thanks for the code review!
This is done now inside [tidy.coeftest](https://github.com/jsr-p/broom/blob/main/R/lmtest-tidiers.R#L58)